### PR TITLE
window on top functionality

### PR DESCRIPTION
**Pull request by PrzemyslawPogorzelec**
Adds option to `XRaiseWindow` instead of `XLowerWindow`.
It can can be used by setting `static unsigned int windowOnTop = 1` in config.h or by adding command line argument `-t`.

### CODE REVIEW

File: stw.1 [hunk 1]
```diff
@@ -66,6 +68,9 @@ period between command runs in seconds
 .BI \-A " alpha"
 window background alpha value
 .TP
+.BI \-t " ontop"
```
Comment (sineemore) [inline stw.1]: You don't need to specify parameter argument ("ontop"), as, IIUC, `-t` is a *bool* parameter.

Check this:

```
default2 ~ st --help
usage: st [-aiv] [-c class] [-f font] [-g geometry] [-n name] [-o file]
          [-T title] [-t title] [-w windowid] [[-e] command [args ...]]
```

Here `st` has several *value* parameters like `-c` and `-f`. They expect a value to be passed.

Also there are bool parameters like `-a` and `-i` at the very beginning of the usage text. They don't expect a value.

___

Remove "ontop" words from usage string and man page and put `-t` at the beginning right after the program name like in `st` man page.

To grasp this topic a bit better check this:
https://pubs.opengroup.org/onlinepubs/9699919799/basedefs/V1_chap12.html

File: stw.c [hunk 2]
```diff
@@ -425,7 +425,10 @@ setup(char *font)
 	gcvalues.graphics_exposures = False;
 	xgc = XCreateGC(dpy, drawable, GCGraphicsExposures, &gcvalues);
 
-	XLowerWindow(dpy, win);
+  if (windowOnTop)
+    XRaiseWindow(dpy, win);
+  else
+    XLowerWindow(dpy, win);
```
Comment (sineemore) [inline stw.c]: Indentation (:

File: stw.c [hunk 3]
```diff
@@ -539,6 +542,9 @@ main(int argc, char *argv[])
 		if (*s == '\0' || *end != '\0' || alpha < 0 || alpha > 1)
 			usage();
 	} break;
+  case 't': {
+    windowOnTop = 1;
+  } break;
```
Comment (sineemore) [inline stw.c]: Also indentation

**Comment by sineemore** [comment]
Btw, just curious, how are you using `ontop` functionality?

**Comment by PrzemyslawPogorzelec** [comment]
I think I fixed it now.

> Btw, just curious, how are you using `ontop` functionality?

I have a script that checks laptop battery level and returns either empty string or message.
The stw window is positioned in such a way that it's invisible when string is empty and visible once message is returned.

I also had some problem with empty strings: changes from empty to non-empty crashed, change non-empty to empty was not seen. I'll open a related issue.

**Pull request merged**